src/components/victory-candlestick/helper-methods.js

```diff
@@ -27,8 +27,8 @@ export default {
       const labelStyle = this.getLabelStyle(style.labels, dataProps);
       const labelProps = {
         style: labelStyle,
-        x,
-        y: y - labelStyle.padding,
+        x: x - labelStyle.padding,
+        y: y - (labelStyle.padding * 2),
```

Comment:
I think this should be something in relation to` y1` if you want to labels to always appear higher than the top of the wick. I think you're just getting lucky by doubling the padding.